mod/class.tx_rnbase_mod_BaseModule.php

```diff
@@ -482,7 +482,10 @@ public function addMessage($message, $title = '', $severity = 0, $storeInSession
 			);
 			$flashMessageService->getMessageQueueByIdentifier()->addMessage($message);
 		} else {
-			$flashMessageClass = tx_rnbase_util_Typo3Classes::getFlashMessageQueueClass();
+			$flashMessageClass = tx_rnbase::makeInstance(
```

Comment:
Actually, the variable does not contain the class (name) anymore, but the instance. So it should be named $flashMessageQueue instead to not be misleading.